core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -47,28 +66,26 @@
 
     // when chunk is frozen, all of the elements in pending puts array will be this OpData
     private static final OpData FROZEN_OP_DATA =
-        new OpData(Operation.NO_OP, 0, 0, 0, null);
+            new OpData(Operation.NO_OP, 0, 0, 0, null);
 
     // defaults
     public static final int MAX_ITEMS_DEFAULT = 4096;
 
     private static final Unsafe unsafe;
     private final MemoryManager memoryManager;
     ByteBuffer minKey;       // minimal key that can be put in this chunk
-    AtomicMarkableReference<Chunk<K,V>> next;
+    AtomicMarkableReference<Chunk<K, V>> next;
     Comparator<Object> comparator;
 
     // in split/compact process, represents parent of split (can be null!)
-    private AtomicReference<Chunk> creator;
+    private AtomicReference<Chunk<K, V>> creator;
```

Comment:
Aren't we done with those <K,V> ?!

Reply:
I did not rebase this branch of the master yet since more code is about to be merged soon.